templates/advanced/api/services/Ember.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Ember service
+ *
+ * @module Ember
+ */
+
+var _ = require( 'lodash' ),
+	pluralize = require( 'pluralize' );
+
+var Ember = {
+	linkAssociations: function ( model, records ) {
+		if ( !Array.isArray( records ) ) records = [ records ];
+		var modelPlural = pluralize( Ember.emberizeModelIdentity( model ) );
+
+		return _.map( records, function ( record ) {
+			var links = {};
+			_.each( model.associations, function ( assoc ) {
+				if ( assoc.type === "collection" ) {
+					links[ assoc.alias ] = sails.config.blueprints.prefix + "/" + modelPlural + "/" + record.id + "/" + assoc.alias;
+				}
+			} );
+			if ( _.size( links ) > 0 ) {
+				record.links = links;
+			}
+			return record;
+		} );
+	},
+
+	emberizeModelIdentity: function ( model ) {
```

Comment:
Now that we are using lodash 3.x we don't need that helper function anymore and could just use https://lodash.com/docs#camelCase